CMakeLists.txt

```diff
@@ -279,6 +279,7 @@ list(APPEND EXPORT_MODULES
     "cmake/AwsSIMD.cmake"
     "cmake/AwsFindPackage.cmake"
     "cmake/AwsFeatureTests.cmake"
+    "cmake/AwsCRuntime.cmake"
```

Comment:
I don't see this function getting called anywhere? Is it called from some dowstream package?

Reply:
Intended for CRT usage really